inventory/sample.gcp.example.com.d/inventory/group_vars/nodes.yaml

```diff
@@ -3,6 +3,10 @@
 openshift_node_labels: 
   region: primary
 
+openshift_node_open_ports:
```

Comment:
@makentenza since `openshift_node_open_ports` is is now defined both in `nodes.yaml` and `infra_hosts.yml`, only one of them will be applied to the infra nodes. I believe you'll need to add the prometheus port to both this file and the existing dict in the  `infra_hosts.yml` as well.

Reply:
@oybed yes, you are right, unless we create a new fact combining both dicts they won't be merged. Will push the changes now